script/clean.py

```diff
@@ -0,0 +1,61 @@
+import csv
+from os import sep
+import unidecode
+import pandas as pd
+
+path = ''
+file = open(path + "amajambo.csv")
+csvreader = csv.reader(file)
+
+rows = []
+for row in csvreader:
+    rows.append(row)
+
+flat_list = []
+for sublist in rows:
+    for item in sublist:
+        flat_list.append(item)
+
+words_list = [item.upper() for item in flat_list]
+
+bad_chars = ['!', "-"]
+
+for i in range(len(words_list)):
+    # remove ascents
+    words_list[i] = unidecode.unidecode(words_list[i])
+    words_list[i] = ''.join((filter(lambda i: i not in bad_chars, words_list[i])))
+
+#save all clean words in csv file
+with open('amajambo_cleanedV1.csv', 'w', newline='') as f:
```

Comment:
You can put the filenames of the output at the beginning of the file. Like this they are easy to change/see.